components/scream/src/share/tests/grid_tests.cpp

```diff
@@ -44,84 +44,28 @@ TEST_CASE("point_grid", "") {
 }
 
 TEST_CASE("se_grid", "") {
-  // Assume a "strip" of elements:
```

Comment:
Is this schematic no longer valid?

Reply:
You mean the test name? Or the  deleted comment? If the latter, I moved it to the mesh free grids manager file, to specify how it creates the SE grid.